app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,3 @@
+@import '../../../node_modules/bootstrap/scss/bootstrap';
```

Comment:
@import '~bootstrap/scss/bootstrap';

Reply:
This line ~bootstrap/scss/bootstrap not working and giving error.May b here is path issue.But @import '../../../node_modules/bootstrap/scss/bootstrap'; working fine.